reflection/Katas/KatasTest/Tests.cs

```diff
@@ -25,12 +25,22 @@ public void Setup()
         public void FizzBuzz_WithVariousInputs_ReturnsExpectedOutput(int number, string expected)
         {
             // Act
-            string result = _fizzbuzz.FizzBuzz(number);
+            string result = _katas.FizzBuzz(number);
             
             // Assert
             Assert.AreEqual(expected, result);
         }
         
+        [Test]
```

Comment:
add at least 2 TestCases with <= 0 and >100 using [TestCase] attribute as you did in other tests